### PR TITLE
Add a contact me form to the saikat-portfolio.

### PR DESCRIPTION
**Pull request by saikatgpae**
This branch is created to add a contact me form to the mobile version.
This is the last task to complete the mobile version of the portfolio project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -197,7 +197,19 @@ <h3 class="am-connect">LET'S CONNECT</h3>
...
+                                <form action="https://formspree.io/f/mbjwvwle" class="contact-form" method="post">
+                                    <ul class="contact-ul">
+                                        <li><input type="text" id="name" required maxlength="30" placeholder="Full Name"></li>
+                                        <li><input type="email" id="email" required placeholder="Email"></li>
+                                        <li><textarea name="text-box" id="text-box" placeholder="Enter text here" required maxlength="500"></textarea></li>
+                                       <li><div class="get-in-touch"><button  type="submit">Get in Touch</button></div></li>
+                                    </ul>
+                                </form>
+                               <hr class="end-line">
+                            </div>
```
Comment (HAMMAS-SALEEM) [inline index.html:212]: ### Your Design
your-fixable-form

### Figma Design
figma-template (copy)

- [x] Please add the following padding in your form shown by the yellow lines in the Figma image above :+1:
- [x] Please make changes to the background image according to the Figma template above image :+1:
- [x] Please make your bar color black and bolder as shown in the Figma template above :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @saikatgpae :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Formspree has been implemented
:heavy_check_mark: Fom validations are working


Good job on implementing some of the requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (saikatgpae) [reply index.html:212]: Changed accordingly.

**Review by muneebulrehman — APPROVED** [review]
Hi @saikatgpae , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by saikatgpae** [comment]
1 page of Contact me added to the work section.

**Pull request merged**